### ISSUE TITLE
includebin seems to not work correctly

### ISSUE THREAD

**Issue by retrotechtive** [Dotneteer/kliveide#748]
```
mc:
	.org #8000
	;	xor a
	
	.org #C736 ; #C737 - this is what the includebin should work with, but does not
	includebin "./colin_mc.bin" ; this is 14536 in length
```

This will compile as it is, though the latter .org has to be one byte lower than it should be (see comment).
If I uncomment the 'xor a' though, it will fail to compile. It's not simply one byte out either, I have to change the second .org to #639B for it to compile (even one byte later at #639C and it fails).

The file I'm including was saved out as binary from FUSE

Thoughts?

**Comment by Dotneteer** [comment]
Hi @retrotechtive, thanks for reporting this issue. I try to reproduce it with another binary file (14536 in length). Of course, if I find the issue, I will fix it ASAP.

**Comment by Dotneteer** [comment]
Hi @retrotechtive, I managed to reproduce the issue. Fix is on the way.

**Comment by Dotneteer** [comment]
Fixed and released in v0.30.5: https://github.com/Dotneteer/kliveide/releases/tag/v0.30.5

**Comment by retrotechtive** [comment]
Fantastic, thanks - great work! :)
